scripts/generate_tilesets.py

```diff
@@ -92,7 +92,8 @@ def get_coco_image_and_segmentations(tile, labels, coco_license_id, coco_categor
 
             try:
                 assert(min(segmentation) >= 0)
-                assert(max(segmentation) <= min(coco_image['width'], coco_image['height']))
+                assert(max(scaled_poly, key = lambda i : i[0])[0] <= coco_image['width'])
```

Comment:
Because the old line says the max coordinate of the labels in the frame of the image should not be bigger than the min between the image width and height.
But, either the image is a square and coco_image['width'] == coco_image['height'] or the image is not a square and the max coordinate can be higher than the min dimension since it can come from the other dimension.
Therefore, I did a code that check the max coordinate on the width and on the height separately, since the tiles I currently use for the roofs are not squares.